### PR TITLE
Added variables to the manual, explained what I could and left blank …

### PR DESCRIPTION
**Pull request by fedepedron**
…what I could not.

### CODE REVIEW

**Comment by ramirezfranciscof** [comment]
Hace otro commit modificando alguna boludez que quiero probar una cosa con los squash.

File: docs/manual/latex/main.tex [hunk 1]
```diff
@@ -81,16 +257,15 @@ \section{Restrains}
...
-    The number of pairs of atoms that going to be added in potential(s) in lio is defined with the variable number\_restr, and the list of distance restrains have to be added to in an extra lio.restrain file like in the following example:
+    \subsection{Using Restraints}
+
+    The number of pairs of atoms to be added in the restraint potential(s) is defined by setting the variable number\_restr, and a list of distance restrains have to be added to in an additional lio.restrain file. For example:
 
     \begin{table}  [H]
       \begin{center}
-      \begin{tabular}{ l c c c c c} 
+      \begin{tabular}{ l c c c c c}
          ai & aj & index &   k  &    wij   &  l0    \\
```
Comment (manuelF) [inline docs/manual/latex/main.tex]: Shouldn't these be $a_i$, $a_j$, $w_{i,j}$ ?

File: docs/manual/latex/main.tex [hunk 2]
```diff
@@ -104,19 +279,17 @@ \section{Restrains}
       \label{lio.restrain}
     \end{table}
 
-In columns ai and aj you find the atom numbers in de QM system to be restrained, index number determine which distances contribute to a same generalized reaction coordinate. Finally k, wij and l0 are the force constant, weight of that distance in the generalized coordinate and equilibrium position in atomic units.
-    
+Columns ai and aj contain the atom numbers in the QM system to be restrained, while the index number determines which distances contribute to a same generalized reaction coordinate. The remaining columns are the force constants (k), weights of that distance in the generalized coordinate (wij) and equilibrium positions in atomic units (l0).
```
Comment (manuelF) [inline docs/manual/latex/main.tex]: Same here for a_i, a_j

**Comment by fedepedron** [comment]
Manuel's corrections -> done.

**Comment by Nick-Chaos** [comment]
A minor modification. there is \label{lio.ecp.var} used twice.

**Comment by fedepedron** [comment]
This pull-merge was way too fast, but we had in mind testing the squash-merge system and this proved useful for that purpose. The manual is by no means finished or whatever, and of course in the future I think this kind of changes should take at least 2-5 days to review (at least depending on the size or complexity).

**Pull request merged**